internal/logger/zap.go

```diff
@@ -0,0 +1,52 @@
+package logger
+
+import (
+	"encoding/json"
+
+	"go.uber.org/zap"
+)
+
+var logger *zap.Logger
+var Sugar *zap.SugaredLogger
```

Comment:
I saw both regular and sugared loggers from zap being used. Shouldn't we use one or the other?